components/Layout/LeadSpace/LayoutLeadSpaceWithCard.vue

```diff
@@ -34,6 +18,22 @@
       </div>
     </div>
   </header>
+  <main class="cds--col layout-lead-space-with-card__main">
```

Comment:
Hi @mariacloehb / @mariahernandezbaeza - this update broke the layout for the event pages we have.

Here's a few examples from your PR's branch preview:

- [Fall Fest](https://qiskit-org-pr-3399.dcq4xc5i083.us-south.codeengine.appdomain.cloud/events/fall-fest)
- [Summer School 2023](https://qiskit-org-pr-3399.dcq4xc5i083.us-south.codeengine.appdomain.cloud/events/summer-school-2023)

Instead of moving everything within the `<main>`, you can also consider just using a `<div>` instead of `<main>`, then renaming the class... 

something like:

```
<div class="cds--col layout-lead-space-with-card__content">
   <!-- rest of the code -->
</div>
```

then in the styles of the same layout, rename `&__main` to `&__content`

Reply:
@techtolentino Just edited it! Let me know if everything is okay